src/python_routing_v02/troute/routing/diffusive_utils.py

```diff
@@ -636,14 +636,16 @@ def diffusive_input_data_v02(
                     
                 idx_segID = np.where(geo_index == segID)
                 iniq[seg, frj] = initial_conditions[idx_segID, 0]
+                if iniq[seg, frj]<0.0001:
```

Comment:
This might optionally be a yaml-specified parameter -- perhaps `minimum_initial_flow: 0.0001`